.github/workflows/package.yml

```diff
@@ -0,0 +1,184 @@
+---
+name: Package CDK
+on:
+  workflow_run:
+    workflows: [Code Health]
+    types:
+      - completed
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    outputs:
+      self_mutation_happened: ${{ steps.self_mutation.outputs.self_mutation_happened }}
+    env:
+      CI: "true"
```

Comment:
[q] is this not set by default? must CIs do it 🤔

Reply:
no, it is not 😢 
projen requires this env when running on CI